### PR TITLE
Korath Exiles Expansion - Stage II 2x Sprites

### PR DESCRIPTION
**Pull request by Saugia**
*Love the content in 1x? Here it is in 2x, again, to match the ships being 2x the size of everything else!*

### CODE REVIEW

**Comment by Zitchas** [comment]
These look beautiful. Looking forward to when they get into the game. :)

**Comment by Zitchas** [comment]
And the main PR is back, so...

**Comment by Zitchas** [comment]
There appears to be a conflict in the copyright file.

File: copyright [hunk 1]
```diff
@@ -1462,7 +1462,7 @@ Copyright: Saugia <https://github.com/Saugia>
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license) and Becca Tommaso (under the same license).
 
-Files:
+Files"
```
Comment (quyykk) [inline copyright:1465]: typo oops

Comment (Saugia) [reply copyright:1465]: o

**Pull request merged**